pom.xml

```diff
@@ -254,19 +255,32 @@
               java.net.http.*
             </ignores>
           </configuration>
-          <executions>
+          <!--<executions>
             <execution>
               <id>ensure-java-1.8-class-library</id>
               <phase>test</phase>
               <goals>
                 <goal>check</goal>
               </goals>
             </execution>
-          </executions>
+          </executions>-->
         </plugin>
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
```

Comment:
Add a comment explaining the presence of this plugin and other spring dependency. It is a non-standard use. 

In fact, please put it in a profile with activation like https://github.com/hub4j/github-api/blob/6cdb1cac5aa29d67b002e7796ae8a4bae2c487c5/pom.xml#L698 and comment the profile.  That will make it on by default but will group these two items together.

Reply:
Because the `AotIntegrationTest` is executed by default with all other tests, the test would fail if you don't execute the plugin by default. Do you think it is really required to put it into a separate profile?

Reply:
No it's fine. I was just thinking of our to encapsulate it. No big deal.